flexmeasures/ui/templates/error.html

```diff
@@ -6,14 +6,14 @@
 
 {% block divs %}
 
-    <div class="container-fluid">
+<div class="card p-3 mt-3 mb-3">
```

Comment:
Can you explain to me shortly what `p-3`, `mt-3` and `mb-3` mean? Why do we have all three of them here?

Reply:
p-3: Adds padding on all sides of the element.
mt-3: Adds margin to the top of the element.
mb-3: Adds margin to the bottom of the element.
These Bootstrap utility classes are used together to ensure the element has padding inside, and margins above and below it for consistent spacing and layout.

Reply:
Thanks!